cmd/gnmi_target/subscribe.go

```diff
@@ -27,8 +30,39 @@ import (
 	pb "github.com/openconfig/gnmi/proto/gnmi"
 )
 
-// eventProducer produces update events for stream subscribers
-func (s *server) eventProducer(dispatcher *dispatcher.Dispatcher,
+// randomEventProducer produces update events for stream subscribers
+func (s *server) randomEventProducer(dispatcher *dispatcher.Dispatcher,
+	subscribe *pb.SubscriptionList) {
+	for {
+		for _, sub := range subscribe.Subscription {
+
+			ipPrefix := "192.168.1"
+			ipSuffix := strconv.Itoa(rand.Intn(254))
+			ip := ipPrefix + "." + ipSuffix
+			subject := "subscribe_stream_randm_event"
+			val := &pb.TypedValue{
```

Comment:
save this val in the map of objects that you keep in memory in order to return it during a get request

Reply:
@Andrea-Campanella @SeanCondon 

I am not sure do we really need to return a value using get or not. Since that feature tries to emulate a change in a predefined path for the testing purposes so that change won't be applied in the config file. So If I want to return it using get that means for that predefined path I should check a condition in the get request to return the value for that predefined path.  Is that what you are suggesting?